### PR TITLE
Updated invoicing packages documentation to include use of the EIA API to retrieve fuel prices

### PR DESCRIPTION
**Pull request by mmayo-truss**

### CODE REVIEW

File: docs/backend/guides/ghc/ghc-invoicing-packages.md [hunk 1]
```diff
@@ -216,50 +216,34 @@ mymove/pkg/services
...
-
-* Add **description** for this package/section.
-* After description is added, filenames (unless it is useful to leave here) can be removed.
-* Leave directory names in section.
-* Be sure to leave filename if the whole package doesn't pertain to invoicing. Leaving only the filenames that
-are important to invoicing. (E.g., `bin` and `cmd` packages. )
-
-:::
-
+The GHC FuelPrice service is utilized by the milmove-task command `save_ghc_fuel_price_data.go`. It calls the EIA API v2.1.0 to retrieve the weekly No. 2 diesel fuel price for the U.S. You can access their [technical documentation here](https://www.eia.gov/opendata/documentation.php). With version 2.1.0 the EIA API added the `seriesid` endpoint. This allows us to use the `series_id` url parameter that we used in the v1 api as a keyword path parameter. The specific path we use is `https://api.eia.gov/v2/seriesid/PET.EMD_EPD2D_PTE_NUS_DPG.W`.
```
Comment (ruizajtruss) [inline docs/backend/guides/ghc/ghc-invoicing-packages.md:221]: The inclusion of a link to their docs is a great move 🎉

**Review by ruizajtruss — APPROVED** [review]
I feel like this is a great overview of not only how we changed our use of the fuel API, but I found that the in depth explanation of the related files was great too, good work on this! 🎉

File: docs/backend/guides/ghc/ghc-invoicing-packages.md (hunk 1 above)
Comment (michaelvaldes-truss) [inline docs/backend/guides/ghc/ghc-invoicing-packages.md:221]: I think one of the most difficult things is about stuff like this is to get a general idea of how/why it's used.
Is it worth adding a sentence or two about how it's run in MilMove (called from an ECS task) and why (to inform service item pricing)? Or is that maybe outside the scope of this documentation?

**Review by michaelvaldes-truss — CHANGES_REQUESTED** [review]

File: docs/backend/guides/ghc/ghc-invoicing-packages.md (hunk 1 above)
Comment (mmayo-truss) [reply docs/backend/guides/ghc/ghc-invoicing-packages.md:221]: That makes sense to me. I'll add it.

**Review by michaelvaldes-truss — APPROVED** [review]
lgtm! Thanks for the extra context

**Pull request merged**